### ISSUE TITLE
error: include of non-modular header inside framework module 'FBSnapshotTestCase.FBSnapshotTestCase'

### ISSUE THREAD

**Issue by knishant90** [facebook/ios-snapshot-test-case#167]
I am trying to integrate the FBSnapshotTestCase with my project and I am facing a build error after I do pod install with the modified podfile. I added the following to my podfile:
`target "xyz" do`
`    pod "FBSnapshotTestCase"`
`end`

The error snapshot that I get in my project after doing a pod install and cmd-B in the project.
![screen shot 2016-06-16 at 3 45 23 pm](https://cloud.githubusercontent.com/assets/6108694/16135592/76fb5dea-33d9-11e6-8291-cdeeedb56235.png)

I dont face this issue if I individually build this project only when I add it in my podfile. Am I doing something wrong or missing something when integrating it with my project?

**Comment by nscoding** [comment]
Can you use cocoapods 1.0.0, the error should disappear

**Comment by knishant90** [comment]
I cant update to 1.0.0, have other pod dependencies which will break. Can I point to an older version of fbsnapshottestcase,will that work?

**Comment by nscoding** [comment]
yes it should, try 2.1.0 with 0.39

**Comment by knishant90** [comment]
I am not able to install "2.1.0" with pod install. My podfile:
`target "InvestorAppTests" do`
    `pod "FBSnapshotTestCase","~>2.1.0"`
`end`

It continues to use 2.1.1
Output:
`Using FBSnapshotTestCase (2.1.1)`

**Comment by nscoding** [comment]
you are right I am not sure what's up with 2.1.0, 2.0.7 works 

```
pod 'FBSnapshotTestCase', '~> 2.0.7'
```

**Comment by knishant90** [comment]
Yeah , but in 2.0.7 I see the issue: #151 when I do a build, I can make the code change to make it work locally but jenkins will not like that. :)

**Comment by knishant90** [comment]
I think the message goes away if I change my podfile to:
`target "InvestorAppTests" do`
    `pod 'FBSnapshotTestCase',:git => 'https://github.com/facebook/ios-snapshot-test-case',:commit => '8e2acac'`
`end`

**Comment by knishant90** [comment]
I was able to resolve it using this in podfile:
`pod 'FBSnapshotTestCase', :git => 'https://github.com/facebook/ios-snapshot-test-case', :commit => '1639f694a2100cdeadf2f8fa14225cdd3759e75b'
`

**Comment by nscoding** [comment]
that's a great tip, thanks for following up.

**Comment by joeblau** [comment]
@knishant90 When I try using that branch, I get the version error:

`[!] The platform of the target 'MyProjectSnapshotTests' (tvOS 9.0) is not compatible with 'FBSnapshotTestCase (2.1.0)', which does not support 'twos'`

**Comment by dhoskins** [comment]
@knishant90 And I get this error

`Test target <MyProjectSnapshotTests> encountered an error (Early unexpected exit, operation never finished bootstrapping - no restart will be attempted) `

**Comment by knishant90** [comment]
@dhoskins try erasing the apps on simulator or Window-> Projects -> delete derived data. This might fix it.

**Comment by knishant90** [comment]
@joeblau : sorry for the late response, were you able to fix it?

**Comment by joeblau** [comment]
I havne't gone back and verified it.  I can check later today or over the weekend and let you know
